frontend/src/components/AddPlayForm.js

```diff
@@ -9,30 +9,44 @@ import PlayersField, { PlayersFieldValidator } from './PlayersField'
 import PlayingTimeField, { PlayingTimeFieldValidator } from './PlayingTimeField'
 import PlayRatingField from './PlayRatingField'
 import { PrimaryButton } from './Button'
+import PlayImageField, { PlayImageFieldValidator } from './PlayImageField.js'
+import { uploadImage, uploadGameData } from '../environment/playDataRestClient'
 
 const AddPlaySchema = Yup.object().shape({
+  playImage: PlayImageFieldValidator,
   gameTitle: GameTitleFieldValidator,
   playDate: PlayDateFieldValidator,
   players: PlayersFieldValidator,
   playingTime: PlayingTimeFieldValidator,
 })
 
-export default function AddPlayForm( { addToPlayCollection }) {
+export default function AddPlayForm({ addToPlayCollection }) {
   return <div>
     <h1>Add a New Play to your Timeline</h1>
+    <p>* Rquired</p>
     <Formik
       initialValues={{
+        playImage: '',
         gameTitle: '',
         playDate: '',
         players: '',
         playingTime: '',
         playRating: 5
       }}
+      validateOnChange={true}
```

Comment:
Bei boolean-Props müsstest du `={true}` weglassen können
```suggestion
      validateOnChange
```

Reply:
Stimmt!